### PR TITLE
SUBMARINE-919. Add develop submarine website guide

### PR DESCRIPTION
**Pull request by pingsutw**
### What is this PR for?

We need to add a  develop submarine website guide in https://submarine.apache.org/docs/devDocs/Development#develop-workbench

- How to build the website
- How to launch the website in local
- How to add a new page (Need to update https://github.com/apache/submarine/blob/master/website/sidebars.js) 

### What type of PR is it?
[Documentation]

### Todos
No

### What is the Jira issue?

https://issues.apache.org/jira/browse/SUBMARINE-919

### How should this be tested?

Pass the CIs.
### Questions:
* Do the license files need updating? No
* Are there breaking changes for older versions? No
* Does this need new documentation? No

### CODE REVIEW

File: website/docs/devDocs/Development.md [hunk 1]
```diff
@@ -146,3 +146,33 @@ Checkstyle plugin may help to detect violations directly from the IDE.
    ```bash
    helm upgrade --set submarine.database.dev=true submarine ./helm-charts/submarine
    ```
+
+## Develop Submarine Website
+Submarine website is built using [Docusaurus 2](https://v2.docusaurus.io/), a modern static website generator.
+
+We store all the website content in markdown format in the `submarine/website/docs`. When committing a new patch to the `submarine` repo, Docusaurus will help us generate the `html` and `javascript` files and push them to  https://github.com/apache/submarine-site/tree/asf-site.
```
Comment (jojochuang) [inline website/docs/devDocs/Development.md:153]: Does Docusaurus run automatically? Or does the committer need to generate the doc and push it up?

Comment (pingsutw) [reply website/docs/devDocs/Development.md:153]: Docusaurus will run automatically in Github action when pushing the commit.

File: website/docs/devDocs/Development.md [hunk 2]
```diff
@@ -146,3 +146,33 @@ Checkstyle plugin may help to detect violations directly from the IDE.
...
+
+If you want to update the website, you can click “Edit this page” on the website.
+
+![](https://lh4.googleusercontent.com/gYcKpxbsGAKv2giTRqkxOehPGnuvnhE31WjsAsYhFmACIZF3Wh2ipar7mZ7F_KRwecM-L1J8YJAgNigJsJUjqc-5IXeO2XGxCIcYpP9CdSc3YByuUkjT_Bezby2HHtkBLyE1ZY_F)
```
Comment (jojochuang) [inline website/docs/devDocs/Development.md:157]: is this an empty ink?

File: website/docs/devDocs/Development.md [hunk 3]
```diff
@@ -146,3 +146,33 @@ Checkstyle plugin may help to detect violations directly from the IDE.
    ```bash
    helm upgrade --set submarine.database.dev=true submarine ./helm-charts/submarine
    ```
+
+## Develop Submarine Website
+Submarine website is built using [Docusaurus 2](https://v2.docusaurus.io/), a modern static website generator.
+
+We store all the website content in markdown format in the `submarine/website/docs`. When committing a new patch to the `submarine` repo, Docusaurus will help us generate the `html` and `javascript` files and push them to  https://github.com/apache/submarine-site/tree/asf-site.
+
+If you want to update the website, you can click “Edit this page” on the website.
```
Comment (jojochuang) [inline website/docs/devDocs/Development.md:155]: "To update the website, click “Edit this page” on the website."

File: website/docs/devDocs/Development.md [hunk 4]
```diff
@@ -146,3 +146,33 @@ Checkstyle plugin may help to detect violations directly from the IDE.
...
+
+![](https://lh4.googleusercontent.com/gYcKpxbsGAKv2giTRqkxOehPGnuvnhE31WjsAsYhFmACIZF3Wh2ipar7mZ7F_KRwecM-L1J8YJAgNigJsJUjqc-5IXeO2XGxCIcYpP9CdSc3YByuUkjT_Bezby2HHtkBLyE1ZY_F)
+
+### Add a new page
+If you want to add a new page to the website, make sure to add the file path to [sidebars.js](https://github.com/apache/submarine/blob/master/website/sidebars.js). 
+
+### Installation
+It's used to install all dependencies for the website.
```
Comment (jojochuang) [inline website/docs/devDocs/Development.md:163]: We use the yarn package manager to install all dependencies for the website

**Review by jojochuang — CHANGES_REQUESTED** [review]
Just a few nits then it's good to go.

File: website/docs/devDocs/Development.md (hunk 2 above)
Comment (pingsutw) [reply website/docs/devDocs/Development.md:157]: It shows the image on the website.
Like below.
image

**Comment by pingsutw** [comment]
Thanks @jojochuang for the review. Updated the doc and fixed the conflict.

**Review by jojochuang — APPROVED** [review]
Ok +1 let's merge it

**Pull request closed without merging**